README.md

```diff
@@ -1,2 +1,252 @@
-# example-repo
-This is a repository to demonstrate the minimal requirements to consider a repo as ready to work with by it's collaborators.
+<a id="readme-top"></a>
+
+<!-- PROJECT SHIELDS -->
+<!--
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![License][license-shield]][license-url]
+
+<!-- PROJECT LOGO -->
+<br />
+<div align="center">
+  <a href="https://github.com/gafda/example-repo">
+    <img src="./docs/images/logo.png" alt="Logo" width="80" height="80">
+  </a>
+
+  <h3 align="center">example-repo</h3>
+
+  <p align="center">
+    An awesome README template to jumpstart your projects!
+    <br />
+    <a href="https://github.com/gafda/example-repo"><strong>Explore the docs »</strong></a>
+    <br />
+    <br />
+    <a href="https://github.com/gafda/example-repo">View Demo</a>
+    ·
+    <a href="https://github.com/gafda/example-repo/issues/new?labels=bug&template=bug-report.md">Report Bug</a>
+    ·
+    <a href="https://github.com/gafda/example-repo/issues/new?labels=enhancement&template=feature-request.md">Request Feature</a>
+  </p>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
+        <li><a href="#built-with">Built With</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#installation">Installation</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#acknowledgments">Acknowledgments</a></li>
+  </ol>
+</details>
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+[![Product Name Screen Shot, may be ani-gif][product-screenshot]](https://example.com)
+
+This project is designed to provide a comprehensive solution for [briefly describe the purpose of the project]. It aims to [mention the main goals or objectives of the project].
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Built With
+
+This section should list any major frameworks/libraries used to bootstrap your project. Leave any add-ons/plugins for the acknowledgements section. Here are a few examples.
+
+* [![dotNetCore][dotnetcore-shield]][dotnetcore-url]
+* [![Bootstrap][bootstrap-shield]][bootstrap-url]
+* [![Docker][docker-shield]][docker-url]
+* [![Kubernetes][kubernetes-shield]][kubernetes-url]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
Put here ### Features
List the main features of the project.

Reply:
Added.